geo_json_metrics/gps_metrics.py

```diff
@@ -128,19 +152,13 @@ def calculate_distance(df: pd.DataFrame):
         df (pd.DataFrame): dataframe to calculate the distances on.
     """
 
-    def shift_elems(l: list):
-        l.pop(-1)
-        return [None] + l
-
-    df["shifted_coordinates"] = df["coordinates"].apply(
-        lambda coordinates: shift_elems(coordinates)
-    )
-
     def calc_dist(row):
```

Comment:
Der måtte godt være en docstring på den her

Reply:
Det er der allerede for shift_elems.